peps/pep-0769.rst

```diff
@@ -1,6 +1,7 @@
 PEP: 769
 Title: Add a 'default' keyword argument to 'attrgetter' and 'itemgetter'
 Author: Facundo Batista <facundo@taniquetil.com.ar>
+Discussions-To: https://discuss.python.org/t/pep-769-add-a-default-keyword-argument-to-attrgetter-and-itemgetter/76419
```

Comment:
Please could you also add `Post-History:`?
```suggestion
Discussions-To: https://discuss.python.org/t/76419
```

Reply:
Improved link and added post-history, thanks!